libs/core/src/lib/tabs/tab-list.component.ts

```diff
@@ -112,18 +134,14 @@ export class TabListComponent implements AfterContentInit, OnChanges, OnDestroy
 
     /** @hidden */
     tabHeaderKeyHandler(index: number, event: any): void {
-        this.tabsService.tabHeaderKeyHandler(index, event, this.tabLinks.map(tab => tab.nativeElement));
+        this._tabsService.tabHeaderKeyHandler(index, event, this.tabLinks.map(tab => tab.nativeElement));
     }
 
-    private isIndexInRange(): boolean {
+    private _isIndexInRange(): boolean {
         return this.panelTabs && this.panelTabs.length > 0 && this.selectedIndex < this.panelTabs.length;
     }
 
-    private isTargetTabEnabled(index: number): boolean {
-        return !this.panelTabs.toArray()[index].disabled;
-    }
-
-    private isTabContentEmpty(): boolean {
+    private _isTabContentEmpty(): boolean {
```

Comment:
Can be simplified to:
```
    private _isTabContentEmpty(): boolean {
        return !this.panelTabs.some(tab => tab.expanded)
    }
```
Also I think method name is misleading it suggests checking if existing tab content has anything inside but in fact it checks if any tab is currently opened.

Reply:
That's true, it's leftover of old code